src/arrays/programs_involving_multidimensional_data/easy/q9/Sum.java

```diff
@@ -0,0 +1,31 @@
+package arrays.programs_involving_multidimensional_data.easy.q9;
+
+/*
+Write a Java program that generates a 2D array of random integers of size 5 by 5, with the range of values from 100-1000
```

Comment:
Change to:
"Write a Java program that generates a 5 by 5 2D array of random integers ranging from 100-1000 (inclusive on both ends), and print the 2D array each row on a new line. Divide all the values by 5, then sum all the values in each row and store those sums into a 1D array which is printed to the console."